src/util/file.h

```diff
@@ -19,6 +19,7 @@
  */
 
 
+int has_perm(int level, struct stat file_stat);
```

Comment:
For new functions, inline documentation in the header would be appreciated (yes, I realize the prior authors didn't...).